tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -235,6 +269,8 @@ def graph_route(self, request):
   @wrappers.Request.application
   def run_metadata_route(self, request):
     """Given a tag and a run, return the session.run() metadata."""
+    if self._data_provider:
+      return None
```

Comment:
Hmm.  Let's send a 404 for now, but really it looks like we should discuss plumbing the run metadata through the data provider.  Looks like it might be a step-indexed blob, so I hope we can hook it up using the existing APIs.  (That would also resolve the weirdness that `is_active()` checks for metadata in the bare multiplexer case, but not in the data provider case).